test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpLogExporterTests.cs

```diff
@@ -44,32 +42,216 @@ public void ToOtlpLogRecordTest()
+            {
+                builder.AddOpenTelemetry(options =>
+                {
+                    options.IncludeFormattedMessage = true;
+                    options.ParseStateValues = true;
+                    options.AddInMemoryExporter(logRecords);
+                });
+            });
+
+            var logger = loggerFactory.CreateLogger("OtlpLogExporterTests");
             logger.LogInformation(new EventId(10, null), "Hello from {name} {price}.", "tomato", 2.99);
             Assert.Single(logRecords);
-            logRecord = logRecords[0];
-            otlpLogRecord = logRecord.ToOtlpLog();
+
+            var logRecord = logRecords[0];
+            var otlpLogRecord = logRecord.ToOtlpLog();
+
             Assert.NotNull(otlpLogRecord);
             Assert.Equal("Hello from tomato 2.99.", otlpLogRecord.Body.StringValue);
+
+            var otlpLogRecordAttributes = otlpLogRecord.Attributes.ToString();
+
+            // Event
+            Assert.Contains("Id", otlpLogRecordAttributes);
+            Assert.Contains("10", otlpLogRecordAttributes);
+
             logRecords.Clear();
 
             logger.LogInformation(new EventId(10, "MyEvent10"), "Hello from {name} {price}.", "tomato", 2.99);
             Assert.Single(logRecords);
+
             logRecord = logRecords[0];
             otlpLogRecord = logRecord.ToOtlpLog();
             Assert.NotNull(otlpLogRecord);
             Assert.Equal("Hello from tomato 2.99.", otlpLogRecord.Body.StringValue);
+
+            otlpLogRecordAttributes = otlpLogRecord.Attributes.ToString();
+
+            // Event
+            Assert.Contains("Id", otlpLogRecordAttributes);
+            Assert.Contains("10", otlpLogRecordAttributes);
+            Assert.Contains("Name", otlpLogRecordAttributes);
+            Assert.Contains("MyEvent10", otlpLogRecordAttributes);
+        }
+
+        [Fact]
+        public void CheckToOtlpLogRecordTraceIdSpanIdFlagWithDroppedActivity()
+        {
+            List<LogRecord> logRecords = new List<LogRecord>();
+            using var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(options =>
+                {
+                    options.AddInMemoryExporter(logRecords);
+                });
+            });
+
+            var logger = loggerFactory.CreateLogger("OtlpLogExporterTests");
+            logger.LogInformation("Log within a dropped activity");
+            var logRecord = logRecords[0];
+            var otlpLogRecord = logRecord.ToOtlpLog();
+
+            Assert.Null(Activity.Current);
+            Assert.True(otlpLogRecord.TraceId.IsEmpty);
+            Assert.True(otlpLogRecord.SpanId.IsEmpty);
+            Assert.True(otlpLogRecord.Flags == 0);
+        }
+
+        [Fact]
+        public void CheckToOtlpLogRecordSpanIdTraceIdAndFlag()
+        {
+            // preparation
+            var sampler = new AlwaysOnSampler();
```

Comment:
Do we need a local variable here or it can be simplified to:
```csharp
            using var tracerProvider = Sdk.CreateTracerProviderBuilder()
                .SetSampler(new AlwaysOnSampler())
                .Build();
```

Reply:
Yup, I think it could be simplified!